analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/GMEFunctionalTest.java

```diff
@@ -50,14 +51,16 @@ public void setup() throws IOException, URISyntaxException {
     }
 
     private void stubDACall() throws IOException, URISyntaxException {
-        stubFor(post(urlEqualTo("/da/rest/v-1/reports/lookup/gavs"))
-                .inScenario("multi-module")
```

Comment:
I've removed the scenario-related lines as they were causing failures and this feature didn't seem to be used anyway.

Reply:
Agreed! Good idea :-)